app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
```

Comment:
Consider using more descriptive variable names for better readability. For example, `command_parts` instead of `commands`.